### PR TITLE
Moves CustomSemanticsProperties to base-ui

### PR DESCRIPTION
**Pull request by fstanis**

### CODE REVIEW

File: audio-ui/src/main/java/com/google/android/horologist/audio/ui/components/actions/SetVolumeButton.kt [hunk 1]
```diff
@@ -30,7 +30,7 @@ import com.google.android.horologist.base.ui.components.IconRtlMode
 /**
  * Button to launch a screen to control the system volume.
  *
- * See [VolumeState]
+ * See [VolumeUiState]
```
Comment (luizgrp) [inline audio-ui/src/main/java/com/google/android/horologist/audio/ui/components/actions/SetVolumeButton.kt:33]: 👍

File: base-ui/src/main/java/com/google/android/horologist/base/ui/components/StandardIcon.kt [hunk 2]
```diff
@@ -21,9 +21,13 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.scale
 import androidx.compose.ui.graphics.vector.ImageVector
 import androidx.compose.ui.platform.LocalLayoutDirection
+import androidx.compose.ui.semantics.semantics
 import androidx.compose.ui.unit.LayoutDirection
 import androidx.wear.compose.material.Icon
+import com.google.android.horologist.annotations.ExperimentalHorologistApi
+import com.google.android.horologist.base.ui.semantics.CustomSemanticsProperties.iconImageVector
 
+@OptIn(ExperimentalHorologistApi::class)
```
Comment (luizgrp) [inline base-ui/src/main/java/com/google/android/horologist/base/ui/components/StandardIcon.kt:30]: nit: not needed if adding the opt in to build.gradle, done in https://github.com/google/horologist/pull/1125

File: base-ui/src/main/java/com/google/android/horologist/base/ui/semantics/CustomSemanticsProperties.kt [hunk 3]
```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.google.android.horologist.audio.ui.semantics
+package com.google.android.horologist.base.ui.semantics
```
Comment (luizgrp) [inline base-ui/src/main/java/com/google/android/horologist/base/ui/semantics/CustomSemanticsProperties.kt:17]: nice! with this change, `CustomSemanticsProperties` from `media-ui` could also be removed

**Review by luizgrp — APPROVED** [review]
LGTM

File: base-ui/src/main/java/com/google/android/horologist/base/ui/semantics/CustomSemanticsProperties.kt (hunk 3 above)
Comment (fstanis) [reply base-ui/src/main/java/com/google/android/horologist/base/ui/semantics/CustomSemanticsProperties.kt:17]: Done.

**Pull request merged**